### PR TITLE
Add sponsor paragraph for Deriv.

### PR DESCRIPTION
**Pull request by thibault-deriv**
Add paragraph for Deriv sponsor!  :tada: :man_dancing: :muscle: :camel: 

sponsor-focus

### CODE REVIEW

File: root/about/sponsors.tx [hunk 1]
```diff
@@ -319,7 +319,23 @@ community.
                 </td>
             </tr>
             <tr>
-                <td><a href="https://kritika.io"><img src="/static/images/sponsors/kritika.svg" class="image-responsive"></a></td>
+                <td><a href="https://deriv.com/careers/"><img src="/static/images/sponsors/deriv-horizontal-web-dark-100.jpeg" width="170" align="left"></a></td>
```
Comment (oalders) [inline root/about/sponsors.tx:322]: Could we add `target="_blank" rel="noopener"` to all of the `href` tags?

**Review by oalders — CHANGES_REQUESTED** [review]

File: root/about/sponsors.tx [hunk 2]
```diff
@@ -319,7 +319,23 @@ community.
...
+                <td>
+                    <p>
+                        <a href="https://deriv.com">Deriv</a> is one of the world’s largest online brokers.
+                        <a href="https://deriv.com">Deriv</a> offers CFDs and other derivatives on forex,
+                        indices, cryptocurrencies, commodities, and synthetics to millions of registered users across the globe.
+                    </p>
+                    <p>
+                        More than that, <a href="https://deriv.com/careers/">Deriv</a> heavily relies on CPAN (and bring back a lot to it in return!)
```
Comment (oalders) [inline root/about/sponsors.tx:330]: This link goes to a different URL, but has the same text as the others. Is this intentional?

Comment (thibault-deriv) [reply root/about/sponsors.tx:330]: Good point I will change this

File: root/about/sponsors.tx (hunk 1 above)
Comment (thibault-deriv) [reply root/about/sponsors.tx]: Done :+1:

File: root/about/sponsors.tx [hunk 3]
```diff
@@ -319,7 +319,23 @@ community.
...
+                <td>
+                    <p>
+                        <a href="https://deriv.com" target="_blank" rel="noopener">Deriv</a> is one of the world’s largest online brokers.
+                        <a href="https://deriv.com" target="_blank" rel="noopener">Deriv</a> offers CFDs and other derivatives on forex,
+                        indices, cryptocurrencies, commodities, and synthetics to millions of registered users across the globe.
+                    </p>
+                    <p>
+                        More than that, <a href="https://deriv.com" target="_blank" rel="noopener">Deriv</a> heavily relies on CPAN (and bring back a lot to it in return!)
+                        with its backend code being written in modern Perl and some great names being part of the company.
+
```
Comment (oalders) [inline root/about/sponsors.tx:332]: There's a merge conflict from your other PR now. ;) We can probably clean up this extra newline at the same time.

**Review by oalders — CHANGES_REQUESTED** [review]

File: root/about/sponsors.tx (hunk 3 above)
Comment (thibault-deriv) [reply root/about/sponsors.tx]: Fixed and fixed :+1:

**Review by oalders — APPROVED** [review]
Thanks, @thibault-deriv! We got this merged eventually. 😄

**Comment by thibaultduponchelle** [comment]
🕺🏼

**Pull request merged**